manuscript.md

```diff
@@ -66,4 +66,22 @@
 
 ## What ecological knowledge would forecasting bring?
 
+
+## Glossary
```

Comment:
I would put this as a box -- markdown has a nice way to write definitions:

~~~
Metaweb
: hella many interactions
~~~

It doesn't work on github markdown, but this should get rendered in the PDF and html